src/components/MetricsPane/MetricsPane.styles.js

```diff
@@ -19,24 +23,20 @@ export const StyledMetricsWrapper = styled.div`
       : css`
           max-width: 40rem;
         `}
-  position: relative;
-  z-index: 5;
-  background-color: ${(props) =>
-    props.$showMobileExpandedMetricsPane ? theme.color.grey1 : 'transparent'};
-  height: 100%;
   ${mediaQueryTabletLandscapeOnly(css`
     position: absolute;
     width: 100%;
     max-width: unset;
+    min-width: unset;
     bottom: 0;
     height: auto;
     ${({ $showMobileExpandedMetricsPane }) =>
       $showMobileExpandedMetricsPane &&
       css`
         top: ${theme.spacing.headerHeight};
-        margin: 0.5rem;
-        margin-bottom: 0;
-        width: calc(100vw - 1rem);
```

Comment:
@alanjleonard I removed the margin that lets us see the map behind the metrics content when its expanded on mobile. In order to make the scroll work, I needed to make a parent div do the actual scrolling, this means when a user scrolls down the charts we can see the map in the background move 'up' and then disappear, so I opted to just remove the margins so we cant see that, and also give us a little more width to work with.